rllib/algorithms/marwil/marwil.py

```diff
@@ -265,10 +264,7 @@ def get_default_rl_module_spec(self) -> RLModuleSpecType:
                 PPOTorchRLModule,
             )
 
-            return RLModuleSpec(
-                module_class=PPOTorchRLModule,
-                catalog_class=MARWILCatalog,
-            )
+            return RLModuleSpec(module_class=PPOTorchRLModule)
```

Comment:
`DefaultPPOTorchRLModule`?

Reply:
Great catch. These are covered, though, by backward compatible imports in the old py-files.

Reply:
changed